modules/gis/test/gis.bench.js

```diff
@@ -3,16 +3,19 @@ import {load} from '@loaders.gl/core';
 
 import {geojsonToBinary} from '@loaders.gl/gis';
 
-const GEOJSON_URL = '@loaders.gl/json/test/data/geojson-big.json';
+// const GEOJSON_URL = '@loaders.gl/json/test/data/geojson-big.json';
```

Comment:
Is there a benefit to changing the benchmark file?

Reply:
The old one didn't have any polygons, so I'm just making it clear there is a perf difference